src/spell.c

```diff
@@ -2250,7 +2250,7 @@ parse_spelllang(win_T *wp)
 	else
 	{
 	    // One entry in 'spellfile'.
-	    copy_option_part(&spf, spf_name, MAXPATHL - 5, ",");
+	    copy_option_part(&spf, spf_name, MAXPATHL - 4, ",");
```

Comment:
📝 The `maxlen` of `copy_option_part()` includes the trailing NUL, so `MAXPATHL - 4` is appropriate.